### PR TITLE
Add sample Gruntfile; Add HBS slide link

### PR DESCRIPTION
**Pull request by JoeShep**
Added another piece in the Gulp to Grunt migration, plus I added a link to my Handlebars intro slide deck to the Handlebars resource file

### CODE REVIEW

File: 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Multi-purpose Gruntfile
+
```
Comment (chortlehoort) [inline 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md]: Can you put a prerequisite in here to make it clear to students that they shouldn't use this Gruntfile until they understand how all the previously discussed, individual tasks work...

[JsHint with watcher](https://github.com/nashville-software-school/front-end-milestones/blob/master/4-modern-javascript-developer/resources/MJ_GRUNT_SETUP.md)
[SASS](https://github.com/nashville-software-school/front-end-milestones/blob/master/4-modern-javascript-developer/resources/MJ_GRUNT_SASS.md)
[Live reload](https://github.com/nashville-software-school/front-end-milestones/blob/master/4-modern-javascript-developer/resources/MJ_GRUNT_LIVERELOAD.md)

**Comment by JoeShep** [comment]
@chortlehoort

File: 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Multi-purpose Gruntfile
 
 ## JSLint, SASS, and Browserify
+_Do not use this Gruntfile until you first have a clear grasp of how each of these individual tasks works:
```
Comment (chortlehoort) [inline 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md]: Remove underline at start of copy.

**Comment by chortlehoort** [comment]
@JoeShep One typo to fix and then merge it.

👍

File: 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md (hunk 1 above)
Comment (studio6thave) [inline 4-modern-javascript-developer/resources/MJ_BROWSERIFY_GRUNTFILE.md]: Scott had pinned a link to htmlhint, can we add that one to the Gruntfile
setup examples?

**Pull request merged**